pom.xml

```diff
@@ -151,6 +151,12 @@
             <version>2.1.17</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>httpclient</artifactId>
+            <version>4.5.12</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
```suggestion
```
This can be removed, can it?

Reply:
Done. I knew there was something I'd forgotten to roll back.